guides/common/modules/proc_installing-standalone-hammer.adoc

```diff
@@ -4,74 +4,124 @@
-ifdef::katello,orcharhino,satellite[]
-For more information, see {ManagingHostsDocURL}Enabling_Custom_Repositories_on_Content_Hosts_managing-hosts[Enabling repositories on hosts] in _{ManagingHostsDocTitle}_.
++
+:distribution-codename: My_Distribution_Codename
+[options="nowrap" subs="verbatim,quotes,attributes"]
+----
+# wget https://deb.theforeman.org/foreman.asc -O /etc/apt/trusted.gpg.d/foreman.asc
+# echo "deb http://deb.theforeman.org/ _{distribution-codename}_ {ProjectVersion}" | sudo tee /etc/apt/sources.list.d/foreman.list
+# echo "deb http://deb.theforeman.org/ plugins {ProjectVersion}" | sudo tee -a /etc/apt/sources.list.d/foreman.list
+----
+. Install Hammer CLI:
++
+[options="nowrap" subs="verbatim,quotes,attributes"]
+----
+# {client-package-install-deb} {project-context}-cli
+----
+. Set the `:host:` entry in the `/etc/hammer/cli.modules.d/foreman.yml` file to the {Project} URL:
++
+[options="nowrap", subs="+quotes,attributes"]
+----
+:host: 'https://_{foreman-example-com}_'
+----
 endif::[]
 ifdef::foreman-el[]
+. Enable the required repositories on the host.
+* If you are installing on {EL}{nbsp}9, enable the following repository:
 +
 [options="nowrap" subs="verbatim,quotes,attributes"]
 ----
-# {client-package-install-el8} https://yum.theforeman.org/releases/{ProjectVersion}/el8/x86_64/foreman-release.rpm
+# {client-package-install-el9} https://yum.theforeman.org/releases/{ProjectVersion}/el9/x86_64/foreman-release.rpm
 ----
-endif::[]
-ifdef::foreman-deb[]
 +
-:distribution-codename: My_Distribution_Codename
+* If you are installing on {EL}{nbsp}8, enable the following repository:
++
 [options="nowrap" subs="verbatim,quotes,attributes"]
 ----
-# wget https://deb.theforeman.org/foreman.asc -O /etc/apt/trusted.gpg.d/foreman.asc
-# echo "deb http://deb.theforeman.org/ _{distribution-codename}_ {ProjectVersion}" | sudo tee /etc/apt/sources.list.d/foreman.list
-# echo "deb http://deb.theforeman.org/ plugins {ProjectVersion}" | sudo tee -a /etc/apt/sources.list.d/foreman.list
+# {client-package-install-el8} https://yum.theforeman.org/releases/{ProjectVersion}/el8/x86_64/foreman-release.rpm
 ----
-endif::[]
-ifndef::foreman-deb[]
-. Enable the following module:
+. If you are installing on {EL}{nbsp}8, enable the following module:
 +
 [options="nowrap" subs="verbatim,quotes,attributes"]
 ----
 # dnf module enable {dnf-module-utils}
 ----
-endif::[]
 . Install Hammer CLI:
 +
 [options="nowrap" subs="verbatim,quotes,attributes"]
 ----
-ifdef::foreman-deb[]
-# {client-package-install-deb} {project-context}-cli
-endif::[]
-ifndef::foreman-deb[]
 # {client-package-install-el8} {project-context}-cli
+----
+. Set the `:host:` entry in the `/etc/hammer/cli.modules.d/foreman.yml` file to the {Project} URL:
++
+[options="nowrap", subs="+quotes,attributes"]
+----
+:host: 'https://_{foreman-example-com}_'
+----
 endif::[]
+ifdef::katello,orcharhino,satellite[]
+. Enable the required repositories on the host.
```

Comment:
I would prefer an unordered substep here in favor of the list of prerequisites. Your call though.

Reply:
I don't think so..? The prerequisite talks about enabling and synchronizing the repositories on the server. This step talks about enabling the repositories for the host on which you are installing Hammer. Those are two different actions.

Now I thought about moving this step to the prerequisites section to keep both actions together but then again, that would make the foreman-deb procedure look very different (because the step for foreman-deb includes a command block).

So, no change, but please, do let me know if you think something about my reasoning feels off.